kernelci.org/content/en/architecture/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "KernelCI Architecture"
+date: 2024-07-03
+description: "Learn the inner details behind the KernelCI systems"
+---
+
+![architecture](/image/kernelci-architecture.png)
+
+The first thing worth noting here is that there are two main parts of the
+overall KernelCI architecture:
+
+### [API & Pipeline](../api_pipeline)
```

Comment:
Shouldn't we refer this as "Maestro"?

Reply:
I was planning to change that as a follow up MR, so we can update the architecture graph, folder name, etc all at once.